main_window/packet_parser.py

```diff
@@ -122,10 +125,7 @@ def header(self, byte_stream: BytesIO) -> Header:
 
         # check that id is valid:
         if not subpacket_ids.isSubpacketID(subpacket_id):
-            print("The current id isn't valid:", subpacket_id)
-            # TODO Error log here?
-            temp = byte_stream.getvalue()
-            raise ValueError
+            LOGGER.exception("Subpacket id %d not valid.", subpacket_id)
```

Comment:
Should be a error instead of exception. Exception is the same as error, it just also grabs the exception's stack trace, message, etc. Not sure what happens if you use this outside of an except block.

Reply:
So I was testing this, and found that LOGGER.exception actually seems to throw an exception itself! That's why I cleaned out the "raise" statements.

Reply:
Reverted to raising exception.